packages/front-end/components/Avatar/Avatar.tsx

```diff
@@ -7,14 +7,31 @@ const Avatar: FC<{
   name: string;
   size?: number;
   className?: string;
-}> = ({ email, size = 40, className, name }) => {
+}> = ({ email, size = 50, className, name }) => {
   const firstNameLetter = name?.charAt(0) || email.charAt(0);
   const lastNameLetter = name?.split(" ")[1]?.charAt(0) || "";
   const title = name ? `${name} <${email}>` : email;
   const copy =
     name.toLowerCase() === "api"
       ? name.toUpperCase()
       : `${firstNameLetter.toUpperCase()}${lastNameLetter?.toUpperCase()}`;
+
+  const getFontSize = () => {
+    const copyLength = copy.length;
+    switch (copyLength) {
+      case 1:
+        console.log(1);
+        return 0.5 * size;
+      case 2:
+        console.log(1);
```

Comment:
Remove console logs

Reply:
just doing a pass over my self then i will let you know once its good